### ISSUE TITLE
Add townofzebulon.org

### ISSUE THREAD

**Issue by jsmozingo** [GSA/govt-urls#97]
Please add townofzebulon.org to the list of recognized local government emails.

**Comment by Cyberamera** [comment]
I agree this should be added in the recognized local government list. This is a public-facing domain for Zebulon town located in the northeast central region of North Carolina.  "The town follows the council-manager form of government, where the elected governing body is responsible for legislative functions such as establishing policy, passing local ordinances, and developing an overall vision. The town's elected board consists of six members (five commissioners and the mayor) who serve for four years term."
https://www.townofzebulon.org/government
https://www.townofzebulon.org/government/board-commissioners
https://data.raleighnc.gov/datasets/Wake::zebulon-zoning/about
